docker/aws/Dockerfile

```diff
@@ -10,3 +10,6 @@ RUN apk update \
     && wget https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -O /usr/local/bin/jq \
     && chmod +x /usr/local/bin/jq
 COPY *.py /
+
+RUN chown jenkins /*py
```

Comment:
Should this also be `/*.py` rather than `/*py`? Otherwise it will include any random files or folders that happen to end in "py".

We should probably move those scripts to a specific folder rather than leaving them in the root directory, but that's more than the scope of this card.

Reply:
Good spot, thanks